doc/source/image_description/elements.rst

```diff
@@ -889,7 +889,15 @@ force_mbr="true|false":
   partitions
 
 gpt_hybrid_mbr="true|false":
-  For GPT disk types only: Create a hybrid GPT/MBR partition table
+  For disk types, create a hybrid GPT/MBR partition table with an
+  'accurate' MBR table that will have no 'bootable' flagged partition
+  For the live ISO type, create a hybrid GPT/MBR partition table
+  where the MBR partition table contains a whole-disk 'protective'
+  partition and a second bootable-flagged partition (intended to
+  make the image bootable in both UEFI and BIOS modes on as much
+  hardware as possible)
+  In both cases, only has any effect if the EFI partition table
+  type is GPT
```

Comment:
```suggestion
  In both cases, this only has any effect if the EFI partition 
  table type is GPT.
```

Reply:
I don't think this is at all necessary, the "this" is implied, the line length is fine, and the file is inconsistent about whether entries end with a period or not. The ones immediately before and after this one don't end with a period either, which is why I did the same for this one.

Reply:
Okay. We really need to take a grammar hammer to the docs... 😅